src/status_im2/contexts/chat/messages/drawers/view.cljs

```diff
@@ -65,13 +69,17 @@
      :style     style/authors-list}]])
 
 (defn reaction-authors
```

Comment:
it seems this file has many components exported. at the very least we should mark the `defn`s that are not intended to be used outside as private to avoid later misuses. Or alternatively it is easier if each file has one exported component (`view`).